src/server/sams/commands/index_from_mongo.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# -*- coding: utf-8; -*-
+#
+# This file is part of SAMS.
+#
+# Copyright 2020 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import time
+import pymongo
+from flask import current_app as app
+
+from superdesk import Command, command, Option, config
+from superdesk.errors import BulkIndexError
+
+from sams.logger import logger
+
+
+class IndexFromMongo(Command):
+    """Index the specified mongo collection in the specified elastic collection/type.
+
+    This will use the default APP mongo DB to read the data and the default Elastic APP index.
+
+    Use ``-f all`` to index all collections.
+
+    Example:
+    ::
+
+        $ python -m sams.manage app:index_from_mongo --from=assets
+        $ python -m sams.manage app:index_from_mongo --all
+
+    """
+
+    option_list = [
+        Option('--from', '-f', dest='collection_name'),
+        Option('--all', action='store_true', dest='all_collections'),
+        Option('--page-size', '-p')
```

Comment:
I think you can add `type=int, default=500` to avoid that extra default_page_size